### ISSUE TITLE
Score Discrepancy About Source

### ISSUE THREAD

**Issue by xzm2002** [saltoricristiano/cosmix-uda#13]
Hello, @saltoricristiano ,

I am currently facing an issue. I am using pre-trained weights and training the model using the command "python train_source.py --config_file configs/source/synlidar2semantickitti.yaml". However, I am getting a score of 27, while the score reported in the paper is 20. I would greatly appreciate any insights you may have as to what could be causing this discrepancy. I was wondering if it is possible that the score reported in the paper was obtained after training the model for only three epochs, as is proposed in adapt_cosmix.py.

**Comment by saltoricristiano** [comment]
Hello @xzm2002 , 

Thanks for the interest in our work.
The 20 in the paper is taken from the paper of SynLiDAR (as they don't provide code for reproducing their numbers) while the 27 is obtained with our warm-up strategy which is part of the CoSMIX framework.
Hope I solved your doubts! I'm going to close the issue but feel free to re-open it!
